docs/using-radicle/creating-projects.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
The flow in this file is not that straightforward. I'd assume a more hands-on guide on how to go through the project creation flow, showing pictures of the modal in the UI explaining what the differences between the two ways are creating a project. Currently the first part tries to explain all the concepts, which might be overwhelming at first and the can be introduced more gently towards the end, when the user lands on the project page aftyer creating their first project.